frontend-vue/src/views/Char_input.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+    <div>I am the character input page</div>
```

Comment:
To be consistent with the other files (e.g. `router.js`), spacing for indents is 2 spaces. You can change how your VSCode does spacing in its bottom toolbar near the right side. Though, changing it isn't retroactive IIRC, so you'll have to change any new files you made (such as these new page files in `views`) to match the spacing.

Reply:
normalized the indentation